### ISSUE TITLE
Running newest Laravel

### ISSUE THREAD

**Issue by zselleristvan** [budisteikul/openshift-cartridge-nginx-php7#2]
Hi, 
Sorry that this question won't be too specific to the stack, but I saw a few words about Laravel in the README, so I've decided to ask, maybe someone can help me. 
So, I install the stack, works awesome. Then I pull the repo, add the Laravel specific config to nginx, plus change the root line to
<code>root    <%= ENV['OPENSHIFT_REPO_DIR'] %>/public/laravel/public; </code>

Then I create a project in  public/laravel folder.
I commit and push. 
I ssh into the app and run <code> composer install</code>. It runs, and I chmod storage to 650. I don't touch .env, because I want to see only the welcoming page at this time. 
I open the URL of the app and it says the app is not working HTTP error 500
Right now I don't know what is the problem  
Error log says:
==> app-root/logs/php-error.log <==
[20-Aug-2016 18:11:45 UTC] PHP Warning:  require(/bootstrap/autoload.php): failed to open stream: No such file or directory in /var/lib/open
shift/57b7405f0c1e66cada000002/app-root/runtime/repo/public/index.php on line 22
[20-Aug-2016 18:11:45 UTC] PHP Fatal error:  require(): Failed opening required '/bootstrap/autoload.php' (include_path='.') in /var/lib/ope
nshift/57b7405f0c1e66cada000002/app-root/runtime/repo/public/index.php on line 22
[20-Aug-2016 18:11:45 UTC] PHP Warning:  require(/bootstrap/autoload.php): failed to open stream: No such file or directory in /var/lib/open
shift/57b7405f0c1e66cada000002/app-root/runtime/repo/public/index.php on line 22
[20-Aug-2016 18:11:45 UTC] PHP Fatal error:  require(): Failed opening required '/bootstrap/autoload.php' (include_path='.') in /var/lib/ope
nshift/57b7405f0c1e66cada000002/app-root/runtime/repo/public/index.php on line 22
[20-Aug-2016 19:10:20 UTC] PHP Warning:  require(/bootstrap/autoload.php): failed to open stream: No such file or directory in /var/lib/open
shift/57b7405f0c1e66cada000002/app-root/runtime/repo/public/index.php on line 22
[20-Aug-2016 19:10:20 UTC] PHP Fatal error:  require(): Failed opening required '/bootstrap/autoload.php' (include_path='.') in /var/lib/ope
nshift/57b7405f0c1e66cada000002/app-root/runtime/repo/public/index.php on line 22
[20-Aug-2016 19:10:20 UTC] PHP Warning:  require(/bootstrap/autoload.php): failed to open stream: No such file or directory in /var/lib/open
shift/57b7405f0c1e66cada000002/app-root/runtime/repo/public/index.php on line 22
[20-Aug-2016 19:10:20 UTC] PHP Fatal error:  require(): Failed opening required '/bootstrap/autoload.php' (include_path='.') in /var/lib/ope
nshift/57b7405f0c1e66cada000002/app-root/runtime/repo/public/index.php on line 22
[20-Aug-2016 19:54:54 UTC] PHP Warning:  require(/bootstrap/autoload.php): failed to open stream: No such file or directory in /var/lib/open
shift/57b7405f0c1e66cada000002/app-root/runtime/repo/public/index.php on line 22
[20-Aug-2016 19:54:54 UTC] PHP Fatal error:  require(): Failed opening required '/bootstrap/autoload.php' (include_path='.') in /var/lib/ope
nshift/57b7405f0c1e66cada000002/app-root/runtime/repo/public/index.php on line 22

Something wrong with the paths, probably. I really need a hand.

**Comment by budisteikul** [comment]
are you sure run `composer install` in right directory?
If you're project on `public/laravel`, you must run `composer install` on `app-root/repo/public/laravel` then make sure root line on nginx configuration in right path. and  don't forget to restart nginx (if you change the nginx configuration), run `service nginx restart` on you're app console

**Comment by zselleristvan** [comment]
Firts of all, I thank you for the quick answer. Still doesn't work, I issued composer install from the directory where composer.json resides, also did service nginx restart when I edited the conf. 
I think this could be a path issue as the log says.  So if I put the laravel installation into the public directory of the stack then the correct path should be public/laravel/public, right? Tried to leave out the last public dir just the heck of it still getting http 500. I'll try to reinstall Laravel and see what will happen.
